Makefile

```diff
@@ -31,7 +31,7 @@ start-executor:
 	@node apps/prairielearn/dist/executor.js
 
 update-database-description:
-	@yarn --cwd apps/prairielearn pg-describe postgres -o ../../database
+	cd apps/prairielearn && yarn pg-describe postgres -o ../../database
```

Comment:
Any reason for this change?

Reply:
For some reason Yarn wasn't picking up the `pg-describe` executable with the `yarn --cwd ...` version. I didn't spend too much time figuring out why. I'm curious, do you see similar behavior locally, or is this something with my environment?

Reply:
This works for me with the `--cwd` option. Another option that works for me is:
```
yarn workspace @prairielearn/prairielearn pg-describe postgres -o ../../database
```

Reply:
That works for me now, huh! I kinda like that better, let me use that instead.